src/ui/src/js/controllers/command_view.js

```diff
@@ -98,6 +102,27 @@ export default function commandViewController(
     return instance.status != 'RUNNING';
   };
 
+  $scope.displayInstanceAlert = function() {
+    let helptext = 'Instance is not RUNNING, ' +
+      'but you can still "Make Request"';
+    const instance = $scope.system.instances[0];
+    if (instance.status == 'AWAITING_SYSTEM') {
+      const requires = $scope.system.requires;
+      const missingSystems = [];
+      requires.forEach((req) => {
+        const system = SystemService.findSystem($scope.system.namespace, req, 'latest');
+        const notRunning = system.instances.every((instance) => instance.status != 'RUNNING');
+        if (notRunning) {
+          missingSystems.push(system.name);
+        }
+      });
+      helptext = 'Unable to determine the status of required system(s): ' +
```

Comment:
Slightly different wording? They might have been found, but just were not running. So we did determine the status of the required system.